From 9410c0a22d07b87e4ab38c35626cbe5c49ec471d Mon Sep 17 00:00:00 2001
From: 
Date: Thu, 16 Jan 2020 18:39:27 -0600
Subject: [PATCH] Rename struct member to fix build failure with musl:

      modules/computer.c:240:27: error: expected identifier before '(' token
        if (detect_lang[i].stdout) {
---
 modules/computer.c | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/modules/computer.c b/modules/computer.c
index cd015cf..13c8022 100644
--- a/modules/computer.c
+++ b/modules/computer.c
@@ -194,7 +194,7 @@ void scan_dev(gboolean reload)
        gchar *compiler_name;
        gchar *version_command;
        gchar *regex;
-       gboolean stdout;
+       gboolean out;
     } detect_lang[] = {
        { "Scripting Languages", NULL, FALSE },
        { "CPython", "python -V", "\\d+\\.\\d+\\.\\d+", FALSE },
@@ -237,7 +237,7 @@ void scan_dev(gboolean reload)
             continue;
        }
        
-       if (detect_lang[i].stdout) {
+       if (detect_lang[i].out) {
             found = g_spawn_command_line_sync(detect_lang[i].version_command, &output, NULL, NULL, NULL);
        } else {
             found = g_spawn_command_line_sync(detect_lang[i].version_command, NULL, &output, NULL, NULL);
-- 
2.25.0

